collector/pkg/model/kindling_event.go

```diff
@@ -305,6 +305,12 @@ func (k *KindlingEvent) GetCtx() *Context {
 	return &k.Ctx
 }
 
+type WorkloadInfo struct {
+	Namespace    string
+	WorkloadName string
+	WorkloadKind string
+}
```

Comment:
Put this definition under the package `kubernetes` since you won't use it in another package.